comments-by-guideline-and-success-criterion.md

```diff
@@ -0,0 +1,1810 @@
+
+See also the discussion on [Closed Functionality](http://w3c.github.io/wcag2ict/#closed-functionality).
+
+Intent from [Understanding Success Criterion 1.2.1 in Understanding WCAG 2.2](http://www.w3.org/WAI/WCAG22/Understanding/audio-only-and-video-only-prerecorded#intent):
+
+> The intent of this Success Criterion is to make information conveyed by prerecorded audio-only and prerecorded video-only content available to all users. Alternatives for time-based media that are text based make information accessible because text can be rendered through any sensory modality (for example, visual, auditory or tactile) to match the needs of the user. In the future, text could also be translated into symbols, sign language or simpler forms of the language (future).
+> 
+> An example of pre-recorded video with no audio information or user interaction is a silent movie. The purpose of the transcript is to provide an equivalent to what is presented visually. For prerecorded video content, authors have the option to provide an audio track. The purpose of the audio alternative is to be an equivalent to the video. This makes it possible for users with and without vision impairment to review content simultaneously. The approach can also make it easier for those with cognitive, language and learning disabilities to understand the content because it would provide parallel presentation.
+> 
+> _Note:_ A text equivalent is not required for audio that is provided as an equivalent for video with no audio information. For example, it is not required to caption video description that is provided as an alternative to a silent movie.
+> 
+> See also _[Understanding Success Criterion 1.2.9 Audio-only (Live)](http://www.w3.org/WAI/WCAG22/Understanding/audio-only-live)_
+> 
+> ###### Specific Benefits of Success Criterion 1.2.1
+> 
+> *   This Success Criterion helps people who have difficulty perceiving visual content. Assistive technology can read text alternatives aloud, present them visually, or convert them to braille.
+>     
+> *   Alternatives for timed-based media that are text based may help some people who have difficulty understanding the meaning of prerecorded video content.
+>     
+> *   People who are deaf, are hard of hearing, or who are having trouble understanding audio information for any reason can read the text presentation. Research is ongoing regarding automatic translation of text into sign language.
+>     
+> *   People who are deaf-blind can read the text in braille.
+>     
+> *   Additionally, text supports the ability to search for non-text content and to repurpose content in a variety of ways.
+>     
+
+Success Criterion 1.2.2: Captions (Prerecorded) (Level A)
+---------------------------------------------------------
```

Comment:
It doesn't seem that headings are being marked down with the pound sign heading markup in several places.